tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSClientTestSupport.java

```diff
@@ -174,13 +174,7 @@ protected String getBrokerCoreJMSConnectionString() {
       try {
          int port = AMQP_PORT;
 
-         String uri = null;
-
-         if (isUseSSL()) {
-            uri = "tcp://127.0.0.1:" + port;
-         } else {
-            uri = "tcp://127.0.0.1:" + port;
-         }
```

Comment:
I assume ErrorProne was complaining that the two legs are the same, and it would be right to. Presumably there are no tests that actually needed / have hit this not being implemented yet, but rather than just remove the logic like this I think it should be left and instead made to throw e.g UnsupportedOperationException / IllegalStateException, so that if anyone later finds themselves in the position of calling this in that situation it is immediately obvious why it wont work and what needs implemented.

Reply:
Fixed.